docker-stack.yml

```diff
@@ -372,3 +376,23 @@ services:
     #         - sauber-network
     #     ports:
     #         - "9990:80"
+
+    db_webhooks:
+        image: "sauberprojekt/db_webhooks:${TAG:-master}"
+        # ports:
+        # volumes:
+        #     - ./nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+        #     - /etc/letsencrypt:/etc/ssl/certs:ro
```

Comment:
Please remove the commented lines, since they seems to copy paste artifacts.

Reply:
Removed placeholders.